OnPod/FlightComputer/src/NodeServer.cpp

```diff
@@ -0,0 +1,125 @@
+#include "FlightComputer/PodInternalNetwork.h"
+#include "FlightComputer/MemoryAccess.h"
+#define UDPPORT 5008
+
+using namespace fc;
+
+void parseBreakNodePacket(fc::brakeNodeData pPayload, MemoryAccess Pod){
+	//Grab State
+	switch (pPayload.state()){
```

Comment:
is this switch irrelevent since each time we'll parse the packet and store the state no matter the state?

Reply:
ya you're right. Im gonna change the type in the pod values struct for PodState to use the enum type make by the protobuf file

Reply:
ya you're right. I'm gonna change the BrakeNodeState var in the pod values to use the enum type made by the protofile. 

Good catch